### ISSUE TITLE
number of very severe exacerbations increases when the model is run again

### ISSUE THREAD

**Issue by aminadibi** [aminadibi/epicR#43]
For example, running this:
```
library(epicR)

for (i in 1:20) {
  init_session()
  run()
  print (Cget_output()$total_exac)
  terminate_session()
  
}

```
will result in:

```
Initializing the session
[1] 3141  561  900   71
Terminating the session
Initializing the session
[1] 2948  548  904  142
Terminating the session
Initializing the session
[1] 2978  502  924  199
Terminating the session
Initializing the session
[1] 2630  486  839  259
Terminating the session
Initializing the session
[1] 2896  585  932  347
Terminating the session
Initializing the session
[1] 2983  499  891  408
Terminating the session
Initializing the session
[1] 2905  545  926  487
Terminating the session
Initializing the session
[1] 3001  552  858  557
Terminating the session
Initializing the session
[1] 2951  496  942  631
Terminating the session
Initializing the session
[1] 3015  553  941  716
Terminating the session
Initializing the session
[1] 2968  494  847  791
Terminating the session
Initializing the session
[1] 2918  558  795  864
Terminating the session
Initializing the session
[1] 2836  542  866  931
Terminating the session
Initializing the session
[1] 3067  505 1025 1002
Terminating the session
Initializing the session
[1] 2875  555  884 1087
Terminating the session
Initializing the session
[1] 3062  539  874 1154
Terminating the session
Initializing the session
[1] 2834  536  919 1226
Terminating the session
Initializing the session
[1] 2860  499  871 1296
Terminating the session
Initializing the session
[1] 2823  490  852 1377
Terminating the session
Initializing the session
[1] 3010  545  967 1464
Terminating the session
```

**Comment by aminadibi** [comment]
Problem must be in the implementation of proportional odds model:

```
  double temp=(*ag).logit_exac_severity_intercept
    +input.exacerbation.logit_severity_betas[3]*(*ag).sex
    +input.exacerbation.logit_severity_betas[4]*((*ag).age_at_creation+(*ag).local_time)
    +input.exacerbation.logit_severity_betas[5]*(*ag).fev1
    +input.exacerbation.logit_severity_betas[6]*(*ag).smoking_status
    +input.exacerbation.logit_severity_betas[7]*(*ag).pack_years
    +input.exacerbation.logit_severity_betas[8]*(*ag).weight/((*ag).height*(*ag).height);

  double l1,l2,l3;
  l1=temp+input.exacerbation.logit_severity_betas[0];
  l2=temp+input.exacerbation.logit_severity_betas[1];
  l3=temp+input.exacerbation.logit_severity_betas[2];
  double p1,p2,p3;

  p1=1/(1+exp(-l1));
  p2=1/(1+exp(-l2))-1/(1+exp(-l1));
  p3=1/(1+exp(-l3))-1/(1+exp(-l2));
// no need for p4, as its value is determined as 1-(p1+p2+p3)

  double r=rand_unif();

  if(r<p1) (*ag).exac_status=1;
  else if(r<(p1+p2)) (*ag).exac_status=2;
  else if(r<(p1+p2+p3)) (*ag).exac_status=3;
  else (*ag).exac_status=4;
```

**Comment by aminadibi** [comment]
No, actually it was the counter that was not reset properly:

```
void reset_output()
{
  output.n_agents=0;
  output.cumul_time=0;
  output.n_deaths=0;
  output.n_COPD=0;
  output.total_pack_years=0;
  output.total_exac[0]=0;output.total_exac[1]=0;output.total_exac[2]=0;;
  output.total_exac_time[0]=0;output.total_exac_time[1]=0;output.total_exac_time[2]=0;
  output.total_doctor_visit[0]=0;output.total_doctor_visit[1]=0;
  output.total_cost=0;
  output.total_qaly=0;
}
```

**Comment by aminadibi** [comment]
Should be fixed now.

**Comment by aminadibi** [comment]
resolved in epicR v0.13.0
